stan/math/prim/mat/fun/matrix_exp.hpp

```diff
@@ -29,6 +29,25 @@ inline Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> matrix_exp(
              ? matrix_exp_2x2(A)
              : matrix_exp_pade(A);
 }
+
+template <typename T, int N>
```

Comment:
We need doxygen comments for every function that gets defined. The comments should specify what the function does (even if this was already done for a previous signature) and the role of each template type and variable. You can specify, if you want, how this signatures differs from the previous signature. See `algebra_solver.hpp` for an example.

Reply:
Thanks. I'll add docs.

Reply:
comments included.